install.bash

```diff
@@ -40,13 +40,13 @@ sudo apt-get install --assume-yes build-essential git python python-scipy sox sw
 # Kaldi install dependencies
 _kaldi_install_dependencies()
 {
+    echo "Checking and installing dependencies..."
     # Change exit to return to source check_dependencies and change back once done
     sed -i "s|exit|return|g" $KALDI/tools/extras/check_dependencies.sh
-    source $KALDI/tools/extras/check_dependencies.sh
+    source $KALDI/tools/extras/check_dependencies.sh > /dev/null 2>&1
```

Comment:
Why would you do `> /dev/null 2>&1` here? I could see why you would do simply `> /dev/null` to suppress the STDOUT, but why would you also want to mute the STDERR here?

Reply:
Valid point. Will make a change there.